lib/datahub-client/data_platform_catalogue/entities.py

```diff
@@ -42,92 +52,54 @@ class FindMoJDataEntityMapper:
     url_formatted: str
 
 
-class TableEntityMapper(FindMoJDataEntityMapper):
-    def __init__(self):
-        super().__init__(
-            "Table",
-            DatahubEntityType.DATASET.value,
-            ["Model", "Table", "Seed", "Source"],
-            "table",
-        )
-
-
-class ChartEntityMapper(FindMoJDataEntityMapper):
-    def __init__(self):
-        super().__init__("Chart", DatahubEntityType.CHART.value, [], "chart")
-
-
-class DatabaseEntityMapper(FindMoJDataEntityMapper):
-    def __init__(self):
-        super().__init__(
-            "Database", DatahubEntityType.CONTAINER.value, ["Database"], "database"
-        )
-
-
-class DashboardEntityMapper(FindMoJDataEntityMapper):
-    def __init__(self):
-        super().__init__(
-            "Dashboard", DatahubEntityType.DASHBOARD.value, [], "dashboard"
-        )
-
-
-class PublicationDatasetEntityMapper(FindMoJDataEntityMapper):
-    def __init__(self):
-        super().__init__(
-            "Publication dataset",
-            DatahubEntityType.DATASET.value,
-            ["Publication dataset"],
-            "publication_dataset",
-        )
-
-
-class PublicationCollectionEntityMapper(FindMoJDataEntityMapper):
-    def __init__(self):
-        super().__init__(
-            "Publication collection",
-            DatahubEntityType.CONTAINER.value,
-            ["Publication collection"],
-            "publication_collection",
-        )
-
-
-class EntityTypes(Enum):
-    """Maps between Find MoJ data's entity type, Datahub entity types and the url_formatted representation
-
-    Each entity has 3 properties:
-     - value: Human-Readable description of the entity
-     - datahub_entity_type: Datahub's description of the entity
-     - url_formatted: URL formatted representation of the entity
-
-     ex: EntityTypes.TABLE.value returns `Table`
-         EntityTypes.GLOSSARY_TERM.datahub_entity_type returns `GLOSSARY_TERM`"""
-
-    TABLE = ("Table", DatahubEntityType.DATASET.value, "table")
-    GLOSSARY_TERM = (
-        "Glossary term",
-        DatahubEntityType.GLOSSARY_TERM.value,
-        "glossary_term",
-    )
-    CHART = ("Chart", DatahubEntityType.CHART.value, "chart")
-    DATABASE = ("Database", DatahubEntityType.CONTAINER.value, "database")
-    DASHBOARD = ("Dashboard", DatahubEntityType.DASHBOARD.value, "dashboard")
-    PUBLICATION_DATASET = (
-        "Publication dataset",
-        DatahubEntityType.DATASET.value,
-        "publication_dataset",
-    )
-    PUBLICATION_COLLECTION = (
-        "Publication collection",
-        DatahubEntityType.CONTAINER.value,
-        "publication_collection",
-    )
-
-    def __new__(cls, value, datahub_entity_type, url_formatted):
-        obj = object.__new__(cls)
-        obj._value_ = value
-        obj.datahub_entity_type = datahub_entity_type
-        obj.url_formatted = url_formatted
-        return obj
+TableEntityMapper = FindMoJDataEntityMapper(
```

Comment:
I wonder if `Mapping` (i.e. a noun) would be a better name than `Mapper` here. Since it's holding data rather than doing the mapping.